src/main/resources/mamute.properties

```diff
@@ -14,8 +14,6 @@ vraptor.errorcontrol.date.joda.pattern = dd/MM/yyyy
 tags.splitter.regex = \\,
 # Char used to split tags inside the JSP forms. It must matchs with the tags.splitter.regex property
 tags.splitter.char = ,
-# Regex of allowed characters for tag names
-tags.sanitizer.regex = [a-zA-Z0-9-]
```

Comment:
I removed it as we are no longer validating the tag names (not needed due
to slugs). Will have to rethink the approach to get c++ and c# working
together though. Thanks for pointing this out.

On Thursday, January 15, 2015, Leonardo Cesar Wolter <
notifications@github.com> wrote:

> In src/main/resources/mamute.properties
> <https://github.com/caelum/mamute/pull/135#discussion-diff-23029896>:
>
> > @@ -14,8 +14,6 @@ vraptor.errorcontrol.date.joda.pattern = dd/MM/yyyy
> >  tags.splitter.regex = \\,
> >  # Char used to split tags inside the JSP forms. It must matchs with the tags.splitter.regex property
> >  tags.splitter.char = ,
> > -# Regex of allowed characters for tag names
> > -tags.sanitizer.regex = [a-zA-Z0-9-]
>
> Sorry, I didn't get it, why did you remove this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/caelum/mamute/pull/135/files#r23029896>.
>


-- 

*Matej Balantič*

T +49 171 2976495
W http://about.me/Matej.Balantic